docs/model-dev-guide/dtrain/dtrain-implement.rst

```diff
@@ -24,110 +25,117 @@ training, as described in :ref:`cluster-configuration`.
+   ``global_batch_size``. ``optimizations.aggregation_frequency`` is useful in scenarios where
+   directly increasing the batch size is not possible (for example, due to GPU memory limitations).
 
 -  ``optimizations.gradient_compression`` reduces the time it takes to transfer gradients between
    GPUs.
 
 -  ``optimizations.auto_tune_tensor_fusion`` automatically identifies the optimal message size
-   during gradient transfers, reducing communication overhead.
+   during gradient transfers, thereby reducing communication overhead.
 
 -  ``optimizations.average_training_metrics`` averages the training metrics across GPUs at the end
-   of every training workload, which requires communication. ``average_training_metrics`` is set to
-   ``true`` by default. This typically does not have a major impact on training performance, but if
-   you have a very small ``scheduling_unit``, disabling this option may improve performance. When
-   disabled, only the training metrics from the chief GPU are reported. This impacts results shown
-   in the WebUI and TensorBoard but does not influence model behavior or hyperparameter search.
+   of every training workload, a process that requires communication. ``average_training_metrics``
+   is set to ``true`` by default and typically does not have a significant impact on training
+   performance. However, if you have a very small ``scheduling_unit``, disabling this option could
+   improve performance. When disabled, only the training metrics from the chief GPU are reported.
+   This impacts results shown in the WebUI and TensorBoard but does not influence model behavior or
+   hyperparameter search.
+
+To learn more about these optimizations, visit the ``optimizations`` section in the
+:ref:`experiment-config-reference`.
 
-If you do not see improved performance using distributed training, there might be a performance
-bottleneck in the model that cannot be directly alleviated by using multiple GPUs, such as with data
-loading. You are encouraged to experiment with a synthetic dataset to verify the performance of
-multi-GPU training.
+If you're not seeing improved performance with distributed training, your model might have a
+performance bottleneck that can't be directly alleviated by using multiple GPUs, such as with data
+loading. You're encouraged to experiment with a synthetic dataset in order to verify the performance
+of multi-GPU training.
 
 .. warning::
 
    Multi-machine distributed training is designed to maximize performance by training with all the
    resources of a machine. This can lead to situations where an experiment is created but never
-   becomes active: if the number of GPUs requested does not divide into the machines available, for
-   instance, or if another experiment is already using some GPUs on a machine.
+   becomes active such as when the number of GPUs requested does not divide into the machines
```

Comment:
There should be a comma in here "becomes active, such as...."

Also, I would slightly rephrase this as "...number of GPUs does not divide **evenly** into the machines..." (though not bold, that's just to make the suggestion stand out for you)